lib/react.dart

```diff
@@ -146,7 +147,7 @@ abstract class Component {
   /// Initializes context
   _initContext(context) {
     this.context = new Map.from(context ?? const {});
-    this.nextContext = this.context;
+    this.nextContext = new Map.from(this.context ?? {});
```

Comment:
#nit this should be a `const` to prevent an extra object allocation
```suggestion
    this.nextContext = new Map.from(this.context ?? const {});
```